src/components/utils/constants.js

```diff
@@ -0,0 +1,15 @@
+export const unitMeasureMap = {
```

Comment:
Could you move these constants to src/utils.js instead of a new file here? We should only have vue components inside /components